docs/_includes/tests/dialog-panel-right.html

```diff
@@ -1,18 +1,8 @@
----
-ds: 4
-layout: test
-title: Right Panel
----
-<svg hidden>
-    <symbol id="icon-close" viewBox="0 0 32 32">
-        <path d="M31.427 2.846l-2.387-2.387-13.084 13.082-13.082-13.082-2.386 2.387 13.082 13.082-13.082 13.084 2.386 2.386 13.082-13.082 13.084 13.082 2.386-2.386-13.084-13.084z"></path>
-    </symbol>
-</svg>
 <div class="test">
     <div role="dialog" class="dialog" id="panel" aria-labelledby="panel-title">
         <div class="dialog__window dialog__window--right" role="document">
-            <button class="dialog__close" type="button">
-                <svg aria-label="Close Dialog">
+            <button class="dialog__close" type="button" aria-label="Close Dialog">
+                <svg class="icon icon--close" aria-hidden="true">
```

Comment:
For my knowledge, what's different between having aria-label on button vs svg?

Reply:
Good question. It's because`aria-label` is much more consistent when it is placed on the focusable/interactive element.

Reply:
It was in the wrong place to begin with.